kremlib/js/loader.js

```diff
@@ -448,7 +448,15 @@ function propagate(module_name, imports, instance) {
 function reserve(mem, size) {
   let m32 = new Uint32Array(mem.buffer);
   let p = m32[0];
-  m32[0] += size;
+  let curr = mem.byteLength;
```

Comment:
Minor: This change seems to be a standalone commit to fix an unrelated issue

Reply:
yes, I need to rebase without #218, good catch, thanks!

Reply:
rebase done